src/components/MainLayout/MainLayout.tsx

```diff
@@ -10,7 +10,7 @@ function Copyright() {
     <Typography variant="body2" color="textSecondary" align="center">
       {"Copyright © "}
       <Link color="inherit" href="https://material-ui.com/" underline="hover">
-        My Store
+        Let&aposs Travel
```

Comment:
This is invalid label. You can use any symbols here as textual data, so:
```
Let's Travel
```
shouldn't be encoded. Please, fix all the places

Reply:
![image](https://user-images.githubusercontent.com/65157909/219956954-dc080140-5a01-4793-aeb5-a871ed8a377b.png)
 
I see eslint error in this case

Reply:
Hmm... Looks strange. OK, let's consider 3 possible ways:
1. change ESLint rules with `"react/no-unescaped-entities": 0`
2. put site name into a constant and use it as JS code without escaping, OR just use `{"Let's Travel"}`
3. use your variant but with correction:
```
Let&apos;s Travel
        ^
you missed semicolon
```
Personally, I would prefer to pick the 2nd variant, but it's up to you. I saw that you made a mistake with missed `;` and decided to dive deeper.

Reply:
Made a quick fix by adding missed ; , because I would like to focus on the backend part. Thank you!